### PR TITLE
Misc linter fixes

### PR DESCRIPTION
**Pull request by oleorhagen**
Fixed a few linter issues I came across.

### CODE REVIEW

**Comment by oleorhagen** [comment]
Based on top of: https://github.com/mendersoftware/mender/pull/1378

@lluiscampos fyi

**Comment by lluiscampos** [comment]
@oleorhagen Thanks! I have fixed few things on my PR, though. You will need to rebase this one once I am done :+1:

File: common/crypto/platform/openssl/crypto.cpp [hunk 1]
```diff
@@ -14,6 +14,14 @@
 
 #include <common/crypto.hpp>
 
+#include <common/crypto/platform/openssl/config.h>
```
Comment (lluiscampos) [inline common/crypto/platform/openssl/crypto.cpp:17]: Probably an error while rebasing. This should be:

```
#include <common/crypto/platform/openssl/openssl_config.h>
```

Which is now few lines below :point_down:

File: common/crypto/platform/openssl/crypto.cpp [hunk 2]
```diff
@@ -241,16 +241,17 @@ expected::ExpectedString EncodeBase64(vector<uint8_t> to_encode) {
...
+	const uint64_t predicted_len {4 * ((to_encode.size() + 2) / 3)};
 
 	// Add space for a NUL terminator. From man page:
 	// Additionally a NUL terminator character will be added
 	auto buffer {vector<unsigned char>(predicted_len + 1)};
 
-	const auto output_len =
-		EVP_EncodeBlock(buffer.data(), to_encode.data(), static_cast<int>(to_encode.size()));
+	const int64_t output_len {
+		EVP_EncodeBlock(buffer.data(), to_encode.data(), static_cast<int>(to_encode.size()))};
```
Comment (lluiscampos) [inline common/crypto/platform/openssl/crypto.cpp:251]: Okey :sob:

**Review by lluiscampos — APPROVED** [review]
Thank you

File: common/crypto/platform/openssl/crypto.cpp (hunk 1 above)
Comment (oleorhagen) [reply common/crypto/platform/openssl/crypto.cpp:17]: Oh, yes! thx!

**Pull request merged**